spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/main/java/com/example/SpannerApplication.java

```diff
@@ -18,11 +18,13 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.gcp.data.spanner.repository.config.EnableSpannerRepositories;
```

Comment:
Not related to this change, but that bumped to my eyes. The package name `com.example` is too broad.
Can't we really have each sample in the proper `org.springframework.cloud.gcp....` package?

This is really our own code, so I feel like we should follow author copyright policy...

Reply:
hmm...I just made the change here, but then realized that all the others are `com.example`...I'd keep it consistent. Maybe create an issue to discuss?

Reply:
Yeah... I'm 100 % for consistency as well, but when it is reasonable.
Let's raise an issue to revise this concern throughout the project!

Sorry, I wasn't clear in my first comment: I exactly wanted to start a discussion on the matter, but not force to do the change in this PR.